File: app/src/main/java/sgtmelon/scriptum/office/st/PageSt.java
package sgtmelon.scriptum.office.st;

/**
 * Состояние для страниц, сохраняющее её номер
 */
public final class PageSt {

    private int page;

    public PageSt() {

    }

    public PageSt(int page) {
        this.page = page;
    }

    public int getPage() {
        return page;
    }

    public void setPage(int page) {
        this.page = page;
    }

}

File: app/src/main/java/sgtmelon/scriptum/office/st/PageSt.kt
package sgtmelon.scriptum.office.st

/**
 * Состояние для страниц, сохраняющее её номер
 */
class PageSt {

    var page: Int = 0

    constructor() {

    }

    constructor(page: Int) {
        this.page = page
    }

}